contracts/oss.yml

```diff
@@ -13237,13 +13237,19 @@ components:
         dropNonRetryableData:
           type: boolean
           default: false
+        maxAge:
```

Comment:
What unit is this in - should we have a comment? And do we usually use seconds or nanoseconds?

Reply:
See for example:

```
    retentionPeriodSeconds:
      type: integer
      format: int64
    retentionPeriodHrs:
      type: integer
      deprecated: true
      description: >
        Retention period *in nanoseconds* for the new bucket. This key's name has been misleading since OSS 2.0 GA,
        please transition to use `retentionPeriodSeconds`
```

Reply:
Considering our retention period is in seconds in might make sense to call this `maxAgeSeconds`